### PR TITLE
Add exists and does not exist string filters

### PR DESCRIPTION
**Pull request by CodeBloodedChris**
On line 193 below I am unable to get around using the equality operator with the None type.
It directly translates to `Model.field IS NULL` on the front end, which is what I need.
`field is None` is simply translated to `False` on the front end. I tried every other way I could think of and can not get the same results as with` field == None`.
I have it currently marked as a Flake8 exception.

### CODE REVIEW

File: src/magql/filter.py [hunk 1]
```diff
@@ -181,6 +187,10 @@ def condition(filter_value, filter_operator, field):
             return field.like(f"%{filter_value}%")
         elif filter_operator == "EQUALS":
             return field == filter_value
+        elif filter_operator == "EXISTS":
+            return field.like("%")
+        elif filter_operator == "DOESNOTEXIST":
+            return field == None  # noqa: E711
```
Comment (Tworley96) [inline src/magql/filter.py:193]: Have you tried `field.is(None)`

Comment (CodeBloodedChris) [reply src/magql/filter.py:193]: I just tried it. Invalid Syntax.
image
Ah, I see I was wrong. It's not evaluated on the front end... I see the graphQL request to the back-end. When I was trouble shooting, if I would print(Model.field == None),  I would see Model.field IS NULL in the terminal, so I was assuming the Python was compiled to JavaScript to be evaluated in browser, but now I see that's not the case.
I'll look into a better solution again today on this one.

Comment (davidism) [reply src/magql/filter.py:193]: The SQLAlchemy `is_()` operator has a trailing underscore to distinguish it from the Python keyword.

Comment (CodeBloodedChris) [reply src/magql/filter.py]: That fixed it. Thanks David!

**Pull request merged**